modules/service_container_doctrine/tests/modules/service_container_doctrine_test/service_container_doctrine_test.services.yml

```diff
@@ -0,0 +1,2 @@
+parameters:
+  annotated_plugin_discovery: {owner: 'service_container_doctrine_test', type: 'Plugin'}
```

Comment:
No this is wrong,

annotated_plugin_discovery: 
  - { owner: 'sc_doctrine_test', type: 'PluginA', directory: 'Plugin/PluginA' }
  - { owner: 'sc_doctrine_test', type: 'PluginB', directory: 'Plugin/PluginB' }